packages/app/components/DialogConfirmTransactionLoading.tsx

```diff
@@ -0,0 +1,25 @@
+import { DotsSpinnerImg } from "@/public/assets";
+import { Dialog, DialogContent, ModalBaseProps } from "@/ui";
+import { ReactNode } from "react";
+
+interface DialogConfirmTransactionLoadingProps
+  extends Omit<ModalBaseProps, "closeAction"> {
```

Comment:
Maybe ModalBaseProps has too much restrictions. Not a problem to fix right now but something we have to be conscious about

Reply:
It's meant to be used on Modals, using it on Dialogs is a workaround as they are similar. 

But we probably need a proper dialog base props. 

We have to be alert on this.